kokoro/builds/build.sh

```diff
@@ -183,12 +183,11 @@ if [ -n "$1" ]; then
 
   # Package the Debian package, the signature and the ggp client into a zip for integration in the 
   # installer.
-  if [ -f ${KEYSTORE_PATH}/74938_SigningPrivateGpg ] && [[ $CONAN_PROFILE == ggp_* ]]; then
+  if [[ $CONAN_PROFILE == ggp_* ]]; then
     pushd "${REPO_ROOT}/build/package" > /dev/null
     mkdir -p Orbit/collector
     cp -v OrbitProfiler*.deb Orbit/collector/
-    cp -v OrbitProfiler*.deb.asc Orbit/collector/
```

Comment:
The .deb.asc file is the signature of the .deb package. We definitely need that one.

Reply:
It was removed temporally because the test was failing, will be included when the temp commits are removed